commands/channel.go

```diff
@@ -345,32 +350,35 @@ func makeChannelPrivateCmdF(c client.Client, cmd *cobra.Command, args []string)
 }
 
 func renameChannelCmdF(c client.Client, cmd *cobra.Command, args []string) error {
-	var newDisplayName, newChannelName string
+	existingTeamChannel := args[0]
+	newChannelName, _ := cmd.Flags().GetString("name")
+	newDisplayName, _ := cmd.Flags().GetString("display_name")
 
-	if len(args) < 2 {
-		return errors.New("Not enough arguments.")
+	// Both Display and name flags are not entered
+	if newDisplayName == "" && newChannelName == "" {
+		return errors.New("Require atleast one flag to rename team, either 'name' or 'display_name'")
 	}
 
-	channel := getChannelFromChannelArg(c, args[0])
+	channel := getChannelFromChannelArg(c, existingTeamChannel)
 	if channel == nil {
-		return errors.New("Unable to find channel '" + args[0] + "'")
+		return errors.New("Unable to find channel '" + existingTeamChannel + "'")
 	}
 
-	newChannelName = args[1]
-	newDisplayName, errdn := cmd.Flags().GetString("display_name")
-	if errdn != nil {
-		return errdn
+	patchedChannel := &model.ChannelPatch{}
```

Comment:
I thought `channelPatch` as the variable name was better. It's not really a "patched" channel because the patch is going to happen and this is the patch. Can you explain your reasoning behind selecting this name ?